### PR TITLE
Migrate and modify Python information

### PR DESCRIPTION
**Pull request by jorgensd**
Documentation was originally at https://github.com/scientificcomputing/reproducibility.
Following #15, we should aim to keep all general documentation on one web page. 

Long-term goal is to make reproducibility a minimal example of how the structure for a Python-package should be. 

I've added new information regarding this layout in `docs/python-packaging.md`

### CODE REVIEW

File: docs/gh-citable.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Make a repository citable
+The easiest way to make your repository citable is to add a `CITATION.cff` file to the repository, as it is supported by both
+[Zenodo's](https://zenodo.org/) and it's [Github integration](https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content) and [Github](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files) itself.
+
+A simple `CITATION.cff` file can look as suggested below:
+```cff
```
Comment (finsberg) [inline docs/gh-citable.md]: I get the following warning: 
```
docs/gh-citable.md:6: WARNING: Pygments lexer name 'cff' is not known
```
Do you have installed any special extension?

File: docs/python-packaging.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+├── README.md
+├── LICENSE
+├── pyproject.toml
+└── src
+    └── name_of_package
+        ├── __init__.py
+        └── functions.py
+```
+### `.git`
+The .git folder is covered in [Version control](./version_control.md)
```
Comment (finsberg) [inline docs/python-packaging.md]: This file does not exist yet

**Review by finsberg — APPROVED** [review]
I think this looks very nice @jorgensd. I added a few changes in order to make the warning go away ;)

File: docs/python-packaging.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+├── README.md
+├── LICENSE
+├── pyproject.toml
+└── src
+    └── name_of_package
+        ├── __init__.py
+        └── functions.py
+```
+### `.git`
+The .git folder is covered in Version control (FIXME: Add link)
```
Comment (jorgensd) [inline docs/python-packaging.md:18]:
```suggestion
The .git folder is covered in [Version control](./version_control)
```

**Pull request merged**